tools/firmware-utils/src/tplink-safeloader.c

```diff
@@ -1930,6 +1930,41 @@ static struct device_info boards[] = {
 		.last_sysupgrade_partition = "file-system",
 	},
 
+    /** Firmware layout for the TL-WR941HP v1 */
+	{
+		.id     = "TL-WR941HP-V1",
+		.vendor = "",
+		.support_list =
+			"SupportList:\n"
+			"{product_name:TL-WR941HP,product_ver:1.0.0,special_id:00000000}\n",
+		.part_trail = 0x00,
+		.soft_ver = NULL,
+
+		/**
+		   384KB were moved from file-system to os-image
+		   in comparison to the stock image
+		*/
```

Comment:
??

Reply:
I remove this lines. I taked this modifications in files from de forum page and fixed leds and buttons part. I thought everything were ok, the router is working well. After some research I changed some things in dts file. Everything is working, the kernel log look normal, please, see it. Can I set a default country for wireless? Thanks for all and sorry for my english. 

[kernel.log](https://github.com/openwrt/openwrt/files/6758681/kernel.log)